fission/function.go

```diff
@@ -165,6 +165,9 @@ func fnCreate(c *cli.Context) error {
 		util.CheckErr(err, fmt.Sprintf("read package in '%v' in Namespace: %s. Package needs to be present in the same namespace as function", pkgName, fnNamespace))
 		pkgMetadata = &pkg.Metadata
 		envName = pkg.Spec.Environment.Name
+		if envName != c.String("env") {
```

Comment:
We should add same code to fnUpdate as well

Reply:
@life1347 - for fnUpdate this is not needed and in fact the behavior there is different but that is for another issue!